sdk/eventhub/azure-eventhub/azure/eventhub/_producer_client.py

```diff
@@ -603,8 +609,11 @@ def send_event(self, event_data, **kwargs):
             else:
                 raise
 
-    def send_batch(self, event_data_batch, **kwargs):
-        # type: (Union[EventDataBatch, SendEventTypes], Any) -> None
+    def send_batch(
+        self,
+        event_data_batch: Union[List[EventData], List[AmqpAnnotatedMessage], EventDataBatch],
```

Comment:
Just want to make sure, it doesn't look like the type hint here matches the docstring:

Union[~azure.eventhub.EventDataBatch, List[Union[~azure.eventhub.EventData,
~azure.eventhub.amqp.AmqpAnnotatedMessage]]

Reply:
Will update!